src/index.jsx

```diff
@@ -48,11 +50,13 @@ export default class GitHubButton extends React.Component {
   }
   getRequestUrl() {
     const { namespace, repo } = this.props;
-    return `//api.github.com/repos/${namespace}/${repo}`;
+    console.log(repo);
+    return `//github.com/${namespace}/${(repo) ? `${repo}/` : ''}`;
```

Comment:
Had you try other types of button? Just as `watchers` `stargazers`? This line of code will break their behavior.